Makefile

```diff
@@ -82,9 +82,6 @@ race_condition_binary: build
 race_condition_run: race_condition_binary
 	./prometheus.race $(ARGUMENTS)
 
-run: binary
```

Comment:
That could work, everything still needs to be updated though. We need to be careful with breaking changes like this, especially as it affects a good chunk of our public docs.